p2p/connection/auth.go

```diff
@@ -0,0 +1,9 @@
+package connection
+
+var whitelist = make(map[string]struct{})
+
+func Whitelist(peers ...string) {
+	for _, peer := range peers {
+		whitelist[peer] = struct{}{}
```

Comment:
It's essentially zero allocation :) since I'm using this map to quickly check if a peer is white listed I don't care about the values under these keys. You will also see it used in channels, when you want to simply use the signal part of them

Reply:
Got it!